requirements/base.txt

```diff
@@ -2,7 +2,8 @@ django==1.8.4
 pillow==2.9.0
 django-stronghold==0.2.7
 django-crispy-forms==1.5.1
-django-authtools==1.2.0
+#django-authtools==1.2.1
+git+https://github.com/fusionbox/django-authtools.git@8f040b9644d0cd2e7694624d9e6f93a0fc5baf0d
```

Comment:
Authtools was using a deprecated api which didn't work with channels. I don't know when 1.2.1 will be released. the maintainer seems to be silent.